src/z3c/autoinclude/utils.py

```diff
@@ -206,3 +215,29 @@ def find_packages(where='.', exclude=()):
 
         out = [item for item in out if not fnmatchcase(item, pat)]
     return out
+
+
+def create_report(info):
+    """Create a report with a list of auto included zcml."""
+    if not info:
+        # Return a comment.  Maybe someone wants to automatically include this
+        # in a zcml file, so make it a proper xml comment.
+        return ["<!-- No zcml files found to include. -->"]
+    report = []
+    # Try to report meta.zcml first.
+    filenames = info.keys()
```

Comment:
`filenames = list(info)` should make this PY3 compatible, for the rest looks amazing :)

Reply:
Done.